shortstories/views.py

```diff
@@ -9,3 +10,16 @@ def index(request):
 def detail(request, shortstory_id):
     shortstory =  get_object_or_404(ShortStory, pk=shortstory_id)
     return render(request,'shortstories/detail.html',{'shortstory':shortstory})
+
+def post_new(request):
+    if request.method == "POST":
+       form = ShortStoryForm(request.POST)
+       if form.is_valid():
+            post = form.save(commit=False)
+            post.author = request.user
+            post.title = request.user
```

Comment:
can you explain why you have this line? `post.title = request.user`

Reply:
It is my bad, need to change that, it is for validation and make sure every field has the correct value, but I wrote wrong one for title. also, need to add for body and define limit for that.